content/en/docs/reference/glossary/static-pod.md

```diff
@@ -2,13 +2,18 @@
 title: Static Pod
 id: static-pod
 date: 2019-02-12
-full_link: /docs/tasks/administer-cluster/static-pod/
+full_link: /docs/tasks/configure-pod-container/static-pod/
 short_description: >
-  A pod managed directly by kubelet daemon on a specific node
+  A pod managed directly by the kubelet daemon on a specific node.
 
 aka: 
 tags:
 - fundamental
 ---
- A {{< glossary_tooltip text="pod" term_id="pod" >}} managed directly by the kubelet
+
+A pod managed directly by the kubelet on a specific node.
+
+<!--more-->
+
+A {{< glossary_tooltip text="pod" term_id="pod" >}} managed directly by the kubelet
```

Comment:
Does this glossary entry need a ` <!--more-->`? The longer entry seems to say almost the same thing as the short version.

(Maybe a _longer_ long version would work instead?)

Reply:
Hi @sftim. That is a nice idea. Here are some current problems:
- The current text for this entry is garbled. I can take some text from the static-pod content page and add to the more section to reduce the duplicated text.
- It would be great if the glossary entries did not have duplicate text; there are a number of entries where this is true.
- All glossary entries use the the hugo `more` summary divider. There appears to be a problem with the processing and rendering of the glossary entries, summary divider. I will log an issue to investigate. I would be in favor of a short term fix 😄